var/spack/repos/builtin/packages/py-pyside2/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class PyPyside2(PythonPackage):
+    """Python bindings for Qt."""
+    homepage = "https://pypi.org/project/PySide2/"
+    url      = "https://code.qt.io/pyside/pyside-setup.git"
+    git      = "https://code.qt.io/pyside/pyside-setup.git"
+
+    # More recent versions of PySide2 (for Qt5) have been taken under
+    # the offical Qt umbrella.  For more information, see:
+    # http://wiki.qt.io/Qt_for_Python_Development_Getting_Started
+
+    version('dev', tag='dev')
+    version('5.13.2', tag='v5.13.2', submodules=True, preferred=True)
+    version('5.13.1', tag='v5.13.1', submodules=True)
+    version('5.13.0', tag='v5.13.0', submodules=True)
+    version('5.12.5', tag='v5.12.5', submodules=True)
+
+    depends_on('python@3.5:', type=('build','run'))
```

Comment:
The `setup.py` says Python 2.7 and 3.5+ are supported

Reply:
The documentation says that the following python version are required

>Python 3.5+ and Python 2.7 (excluding Python 3.6.0)

I read this as python `2.7` (exactly), `3.5.0:3.5.99`, or `3.6.1:`.  How do I express this in my `depends_on` command?

Reply:
Does this look right: `depends_on('python@2.7:2.7.99, 3.5.0:3.5.99, 3.6.1:')`